logger/logger_writer_test.go

```diff
@@ -0,0 +1,61 @@
+package logger
+
+import (
+	"fmt"
+	"io"
+	"math/rand"
+	"runtime"
+	"testing"
+	"time"
+)
+
+func TestReleaseMemory(t *testing.T) {
```

Comment:
Not a blocker but this test feels like more of a development/debug tool than an actual test.  I'm not opposed to checking it in but do you think we should disable or skip it?

Reply:
You're totally right.  I'm leaving it in place, but skipping it, so it's there the next time we need to confirm a similar issue.